external/builder/fixtures_babel/evals.js

```diff
@@ -5,7 +5,6 @@ var d = PDFJSDev.test('FALSE');
 var e = PDFJSDev.eval('TRUE');
 var f = PDFJSDev.eval('TEXT');
 var g = PDFJSDev.eval('OBJ');
-var h = PDFJSDev.json('$ROOT/external/builder/fixtures_babel/evals.json');
```

Comment:
Does this makes that file unused?
If so, we should also remove `external/builder/fixtures_babel/evals.json` as part of this patch.